pkg/cli/sql.go

```diff
@@ -540,6 +546,22 @@ func (c *cliState) handleDemo(cmd []string, nextState, errState cliStateEnum) cl
 		return c.invalidSyntax(errState, `\demo expects 2 parameters`)
 	}
 
+	// Special case the add command it takes a string instead of a node number
+	if cmd[0] == "add" {
```

Comment:
optional personal nit: would prefer all the cmd[0]s to be switched in the same "area", maybe something like:

```
switch cmd[0] {
   case "add":
     ....
   case "recommission", "decommission", ...:
     <handle parse node id, etc, or maybe put this entire section into a new function
     switch cmd[0] {
        case "recommission": // ....
      }
}

```

so we're upfront about all possible states and cmd[0] is in one place. also i anticipate `add` taking more arguments in future (i imagined at a later stage we might want this such that `\demo add` would take the same flags as `cockroach start`) and the len(cmd) != 2 check may need to be moved lower.